ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -506,11 +506,9 @@ export default defineComponent({
         return props.setItem
       }),
       enableOpenEdit: computed(() => {
-        return isRoleQualifiedSupplier.value &&
-          !isRoleStaff.value &&
+        return (isRoleQualifiedSupplier.value || isRoleStaffReg.value || isRoleStaff.value) &&
           !isRoleStaffSbc.value &&
```

Comment:
No action required here, oddly enough this check is required because `isRoleStaff` includes SBC Staff.
Would be scary to change something like this now but just what i noticed recently in other work.

Reply:
ok